influxdb3/batching/batcher.go

```diff
@@ -24,43 +24,72 @@ THE SOFTWARE.
 package batching
 
 import (
+	"fmt"
+	"log/slog"
 	"sync"
 
 	"github.com/InfluxCommunity/influxdb3-go/influxdb3"
 )
 
 // Option to adapt properties of a batcher
-type Option func(*Batcher)
+type Option func(*interface{})
```

Comment:
Use `any` instead of `interface{}`

Reply:
Refactored to 
```go
type Option func(BatcherIface)
```
Now in file `batcher_iface.go`

Reply:
Now separated out to two types of options. Original `Option` works with `PointEmittable`

```go
type Option func(PointEmittable)
```
New `LPOption` works with `ByteEmittable`
```go
type LPOption func(ByteEmittable)
```
It's taking some time to start thinking in _GO_ and to quit trying to force _GO_ to behave like other languages.